opencga-analysis/src/main/java/org/opencb/opencga/analysis/tools/OpenCgaTool.java

```diff
@@ -495,6 +499,20 @@ protected final void addAttribute(String key, Object value) throws ToolException
         erm.addAttribute(key, value);
     }
 
+    protected final void addStepAttributes(ExecutionResult executionResult) throws ToolException {
```

Comment:
You might want to include more information from the nested ExecutionResult, like events (maybe that's the only interesting thing)

Reply:
do you mean to add the "nested" ExecutionResult events into the list of "parent" Execution events?,

Reply:
Yes, something like that. Otherwise, they're lost (not technically, as you can always read the nested job result file, but they'd be quite hidden).